### ISSUE TITLE
We should finish (and start distributing) the DoES Liverpool New Member Manual

### ISSUE THREAD

**Issue by amcewen** [DoESLiverpool/somebody-should#118]
The idea was to develop something like https://wiki.nottinghack.org.uk/images/9/96/Nottingham-hackspace-manual-web-Nov2011.pdf

Andy Goodwin did an initial draft, which I still haven't looked through, but we should at least be tracking progress on it in here :-)

**Comment by DefProc** [comment]
Just pushed the content to https://github.com/DoESLiverpool/Handbook as markdown

**Comment by DefProc** [comment]
From #111, the welcome pack needs to include the new [newdinky.youcanbook.me](http://newdinky.youcanbook.me) link in it

**Comment by amcewen** [comment]
As mentioned in #125, we should explain where the bins should be emptied, the recycling systems, and where bulkier (workshop) waste needs to go.

**Comment by amcewen** [comment]
Copying my comment over from #434...

The handbook is, as you rightly surmise, the "introduction to DoES" guide.  There will be a massive overlap with the wiki, and it'd be good to get to a point where it pulls in chunks of the wiki (so we only have to write things once) and similarly directs people to the wiki.  I think it'd also have some more background info on DoES as a community, and have the subset of info that's most important to a newcomer.  (And likely be created as a PDF we can email to them, or maybe we'd even get some copies printed up - there's been talk of it becoming something akin to the [Hiut Denim yearbook](http://hiutdenim.co.uk/products/year-book-3), with interesting articles interspersed with the need-to-know info.

**Comment by amcewen** [comment]
I wonder if there's an easy way to add some markers to certain bits of content in the wiki so that we can write a script to generate the handbook which can pull in the relevant bits from the wiki?

**Comment by aubergine10** [comment]
@amcewen Yes, it's possible - the ESLint project would be a good example and code reference. There are markdown linters which can help validate content prior to exporting, and then converters to turn it in to HTML, PDF, etc. If you need more infos let me know.

I've just been reading through the handbook and many sections are incomplete or out of date. I'll make some quick changes to fix broken links (eg. to the old wiki) but I'm not sure I can currently help much with the rest of it as I'm still a DoES newbie.

Suggestion: Wait until the updated "DoES vision" (marketing, etc) is taking shape before doing any more work on handbook (as the handbook should be align with 'marketing' of DoES and vice versa).

**Comment by amcewen** [comment]
I think @aubergine10's suggestion of waiting until the vision, etc. has been done is a good one.  This will be moving up my priority list once the vision stuff is finished.

And the main reason for this comment is to add a link to the [Institute of Making's 4th yearly report](http://www.instituteofmaking.org.uk/assets/_files/uploads/Fourth_Year_Report_web.pdf) as another example of something to crib ideas from.

I particularly like the "members' profiles" section, but some of the "talks given" and "projects undertaken" are nice ideas too (maybe "projects undertaken" could be more "notable collaborations", to highlight how people end up building things together - Tosca, Fomocam...)

**Comment by RiverMersey** [comment]
Could this thread / issue be closed again now?

**Comment by DoESsean** [comment]
Not yet, as we don't have the member booklet created yet.

It's been a long time coming, and does seem like it may never happen, but we're not giving up on it entirely just yet!

**Comment by JackiePease** [comment]
Another example: [Bristol Hackspace Manual](https://wiki.bristolhackspace.org/manual/home)
